### ISSUE TITLE
Routing on Nav icon and transform size of social handles in footer

### ISSUE THREAD

**Issue by hsnice16** [CodeTrophs/Frontend#329]
Right now, on the hovering nav icon, the cursor is changing to pointer but on clicking it, nothing happens. So I think on clicking it route should change to '/' (or home page).

And in the footer, on hovering social handlers only the cursor is changing to a pointer type, if we also transform its size then it will look nicer.

**Comment by Anurag5086** [comment]
@hsnice16 Go ahead

**Comment by hsnice16** [comment]
Please guide me, I am new to next.
Will this work correctly? I am trying to make nav icon a link.
```
<Link href='/feed'>
      <a>
      <div className={styles.logo}>
      <img className= {styles['header-logo']} src="/logo/codetrophs.png" alt="" />
      </div>
      </a>
</Link>
```

**Comment by aryasoni98** [comment]
@Anurag5086 Review

**Comment by Anurag5086** [comment]
> Please guide me, I am new to next.
> Will this work correctly? I am trying to make nav icon a link.
> 
> ```
> <Link href='/feed'>
>       <a>
>       <div className={styles.logo}>
>       <img className= {styles['header-logo']} src="/logo/codetrophs.png" alt="" />
>       </div>
>       </a>
> </Link>
> ```

Yeah this looks good. Make the code properly indented.

**Comment by hsnice16** [comment]
Sure

**Comment by hsnice16** [comment]
Please close this issue.

**Comment by msk4862** [comment]
Closes via #339
